base/stream.jl

```diff
@@ -1098,6 +1098,12 @@ for (x, writable, unix_fd, c_symbol) in
             ($f)($(writable ? :write : :read))
             return (read, write)
         end
+        function ($f)(::DevNull)
+            nulldev = @static Sys.iswindows() ? "NUL" : "/dev/null"
+            handle = open(nulldev, write=$writable)
+            $(_f)(handle)
+            return handle
+        end
```

Comment:
Would `handle` be closed after `redirect_stdout(f, devnull)`? (It looks like not?)

Reply:
You are right - I am working to close the handle when it is no longer needed.